pkg/controllers/dynakube/extension/conditions.go

```diff
@@ -0,0 +1,32 @@
+package extension
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func setSecretCreatedSuccessCondition(conditions *[]metav1.Condition) {
+	condition := metav1.Condition{
+		Type:    secretConditionType,
+		Status:  metav1.ConditionTrue,
+		Reason:  secretCreatedReason,
+		Message: secretCreatedMessageSuccess,
```

Comment:
same here, or move it to top of this file like in https://github.com/dynatrace/dynatrace-operator/blob/2a85bd2f4aded19b83f259c00fc6f645b4b6e1cd/pkg/controllers/dynakube/injection/conditions.go#L17